packages/database/src/repository/telegram.ts

```
@@ -0,0 +1,22 @@
+    const [binding] = await useDatabase().insert(telegramBindings).values(data).returning()
+    return binding
+  }
+
+  static async createReceiver(data: TelegramReceiverDraft) {
+    const [receiver] = await useDatabase()
+      .insert(telegramReceivers)
+      .values(data)
+      .returning()
+    return receiver
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)